### DISCUSSION TITLE
Two spaces after timestamp

### DISCUSSION THREAD

**Discussion by willettjf** [YU000jp/logseq-plugin-confirmation-done-task#12] [Q&A]
Can I change this behavior to only insert one space after the timestamp?
<img width="202" alt="Screenshot 2023-06-25 at 10 01 26 AM" src="https://github.com/YU000jp/logseq-plugin-confirmation-done-task/assets/2234398/b9310172-8545-4ed6-92e3-3be39c71912b">

**Comment by YU000jp** [comment]
It's not about the asterisk, of course the space, right? I didn't intend to place a space after the timestamp. I believe it might be a bug, as it's not behaving as intended. If it's possible to consider an alternative behavior that aligns with your workflow, I'd be willing to explore that option.

**Reply by willettjf** [reply]
I like bold. It would be perfect with just one space after time stamp. It's currently giving me two (maybe bug?), and so I have to backspace before typing. Also a chance I'm doing something wrong so maybe see if you can replicate.

**Reply by YU000jp** [reply]
I have released a new version. Does it meet our intended expectations?

**Comment by willettjf** [comment]
Yes thanks!
